lib/taiko.js

```diff
@@ -3121,6 +3122,16 @@ getPlugins().forEach(pluginName => {
   }
 });
 
+/**
+ * Lets you read the global configurations.
+ *
+ * @example
+ * getConfig("retryInterval");
```

Comment:
Can you please add the other config param - 'navigationTimeout, observeTime, retryInterval, retryTimeout, observe, waitForNavigation, ignoreSSLErrors, headful, highlightOnAction';

Reply:
Is this what you had in mind?